content/ko/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: blog
+title: "쿠버네티스 1.24: gRPC 컨테이너 프로브 베타"
+date: 2022-05-13
+slug: grpc-probes-now-in-beta
+---
+
+**Author**: Sergey Kanzhelev (Google)
+
+
+Kubernetes 1.24에서 gRPC 프로브 기능이 베타에 진입했으며 기본적으로 사용 가능합니다.
+이제 HTTP 엔드포인트를 노출하지 않고, gRPC 앱에 대한 스타트업, 활성 및 준비성 프로브를 구성할 수 있으며, 실행 파일도 필요하지 않습니다. Kubernetes는 기본적으로 gRPC를 통해 워크로드에 연결하고 상태를 쿼리할 수 있습니다.
+
+## 약간의 히스토리
+
+시스템이 워크로드의 앱이 정상인지, 정상적으로 시작되었는지, 트래픽을 수용할 수 있는지에 대해 관리하는 것은 유용합니다. gRPC 지원이 추가되기 전, Kubernetes는 이미 컨테이너 이미지 내부에서 실행 파일을 실행하거나, HTTP 요청을 하거나, TCP 연결이 성공했는지 여부를 확인하여 상태를 확인할 수 있었습니다.
+
+대부분의 앱에서, 이러한 검사로 충분합니다. 앱이 상태(또는 준비) 확인을 위한 gRPC 엔드포인트를 제공하는 경우 exec 프로브를 gRPC 상태 확인에 사용하도록 쉽게 용도를 변경할 수 있습니다.
+블로그 기사 [Kubernetes의 gRPC 서버 상태 확인](/blog/2018/10/01/health-checking-grpc-servers-on-kubernetes/)에서, Ahmet Alp Balkan은 이를 수행하는 방법을 설명했습니다. 이는 지금도 여전히 작동하는 메커니즘입니다.
+
+2018년 8월 21일에 [생성](https://github.com/grpc-ecosystem/grpc-health-probe/commit/2df4478982e95c9a57d5fe3f555667f4365c025d)되었으며, [2018년 9월 19일](https://github.com/grpc-ecosystem/grpc-health-probe/releases/tag/v0.1.0-alpha.1)에 첫 번째 릴리스와 함께 이를 활성화하는데 일반적으로 사용되는 도구가 있습니다.
+
+gRPC 앱 상태 확인을 위한 이 접근 방식은 매우 인기있습니다. `grpc_health_probe`가 있는 [Dockerfile은 3,626개](https://github.com/search?l=Dockerfile&q=grpc_health_probe&type=code), GitHub(작성 시점)의 기본 검색으로 발견된 [yaml 파일은 6,621개](https://github.com/search?l=YAML&q=grpc_health_probe&type=Code) 있습니다. 이것은 도구의 인기와 이를 기본적으로 지원해야 할 필요성을 잘 나타냅니다.
+
+Kubernetes v1.23은 gRPC를 사용하여 워크로드 상태를 쿼리하기 위한 알파 품질의 구현을 기본 지원으로 도입했습니다. 알파 기능이었기 때문에 v1.23 릴리스에서는 기본적으로 비활성화되었습니다.
+
+## 기능 사용
+
+우리는 다른 프로브와 유사한 방식으로 gRPC 상태를 확인할 수 있도록 구축했으며, Kubernetes의 다른 프로브에 익숙하다면 [사용하기 쉬울 것](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)이라 믿습니다.
```

Comment:
```suggestion
우리는 다른 프로브와 유사한 방식으로 gRPC 상태를 확인할 수 있도록 구축했으며, 쿠버네티스의 다른 프로브 타입들에 익숙하다면 [사용하기 쉬울 것](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe)이라 믿습니다.
```
원문과 유사하게 `프로브 타입들`로 수정해보았습니다 :)

```
We built gRPC health checking in similar way with other probes and believe it will be easy to use if you are familiar with other probe types in Kubernetes.
```